src/modelinstance.cpp

```diff
@@ -140,6 +117,35 @@ Status ModelInstance::loadInputTensors(const ModelConfig& config, const DynamicM
         SPDLOG_DEBUG("model: {}, version: {}; reshaping inputs is not required", getName(), getVersion());
     }
 
+    for (const auto& pair : networkInputs) {
```

Comment:
In case of loading error we will be missing logs with precision/layout of inputs from OV which is usually helpful.